### DISCUSSION TITLE
ERROR when trying to run Nextflow for RNAseq on MacOSX (M2)

### DISCUSSION THREAD

**Discussion by ag4742** [nextflow-io/nextflow#3676] [Q&A]
## ERROR

`$ nextflow run nf-core/rnaseq -profile test,conda --outdir /Users/aggardik/workflow N E X T F L O W ~ version 0.30.1 Launching nf-core/rnaseq [jovial_nightingale] - revision: 6e1e448f53 [master] ERROR ~ Unknown config attribute params.genomes.GRCh37.projectDir -- check config file: /Users/aggardik/anaconda3/envs/RNAseq/share/nextflow/assets/nf-core/rnaseq/nextflow.config -- Check '.nextflow.log' file for details (RNAseq)`

I am getting this right when I try to download the pipeline and test it on a minimal dataset with a single command.

* Nextflow version: version 0.30.1 
* Java version: openjdk version "1.8.0_152-release"
* Operating system: macOS (M2 2023)
* Bash version: GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22)

**Comment by colindaven** [comment]
It would probably be best to ask this on the nf-core rna-seq repository, since this repo is for nextflow itself, not nf-core.
